runtime/kusama/src/lib.rs

```diff
@@ -1474,15 +1473,13 @@ construct_runtime! {
 		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 2,
 		Indices: pallet_indices::{Pallet, Call, Storage, Config<T>, Event<T>} = 3,
 		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 4,
-		TransactionPayment: pallet_transaction_payment::{Pallet, Storage, Event<T>} = 33,
```

Comment:
The ordering of pallets here was intentional. I think the goal is to group pallets together of a similar kind, even if their indexes are not in order.